src/Facility.CodeGen.CSharp/CSharpGenerator.cs

```diff
@@ -278,17 +292,6 @@ private CodeGenFile GenerateEnum(ServiceEnumInfo enumInfo, Context context)
 							}
 						}
 
-						code.WriteLine();
-						code.WriteLine($"private static readonly Dictionary<string, string> s_normalizedConstants = new Dictionary<string, string>");
```

Comment:
This seems right, except it needs `(StringComparer.OrdinalIgnoreCase)`. Why this change?

Reply:
It was a half-baked idea, but the rationale was `m_value` would always be normalized (which I get doesn't make sense now).